onadata/apps/logger/models/note.py

```diff
@@ -15,10 +22,17 @@ class Note(models.Model):
     date_modified = models.DateTimeField(auto_now=True)
 
     class Meta:
+        """
+        Meta Options for Note Model
+        """
         app_label = 'logger'
         permissions = (('view_note', 'View note'), )
 
     def get_data(self):
+        """
+        Custom Method that returns a Dict Containing
```

Comment:
Could be "Returns Note data as a dictionary."

Reply:
Updated the Docstring on #1455